### PR TITLE
mod: adjust conquest rewards to be fair for both teams

### PR DESCRIPTION
**Pull request by MrMendo**
As people many times saids that conquest doesn't give fair rewards to attackers i adjusted the behaviour of the multiplier as shown in this concept:
https://docs.google.com/spreadsheets/d/1IzlttmcPsHRfWwRVzOeCJ9wGHe7ghe3ZvzbN54MLecY/edit#gid=1602412326

Please note that the tick rate is supposed to be 30seconds. So that needs to be adjusted aswell after patching.

### CODE REVIEW

File: src/Module.Server/Modes/Conquest/CrpgConquestServer.cs [hunk 1]
```diff
@@ -45,6 +46,20 @@ public override MissionLobbyComponent.MultiplayerGameType GetMissionType()
...
+        if (crpgPeer?.User == null)
+        {
+            return;
+        }
+
+        if (oldTeam == null && newTeam.Side == BattleSideEnum.Attacker)
+        {
+            crpgPeer.RewardMultiplier = Math.Max(crpgPeer.RewardMultiplier, 3);
+        }
+    }
```
Comment (verdie-g) [inline src/Module.Server/Modes/Conquest/CrpgConquestServer.cs:61]: Why is that needed?

File: src/Module.Server/Modes/Conquest/CrpgConquestServer.cs [hunk 2]
```diff
@@ -417,10 +434,25 @@ private void RewardUsers()
...
+                defenderMultiplierGain = 0;
+                attackerMultiplierGain = 0;
+                _isOddRewardTick = false;
+            }
+            else
+            {
+                defenderMultiplierGain = 1;
+                attackerMultiplierGain = -1;
+                _isOddRewardTick = true;
+            }
```
Comment (verdie-g) [inline src/Module.Server/Modes/Conquest/CrpgConquestServer.cs:450]: It sounds like you could just double the tick interval right?

**Review by verdie-g — CHANGES_REQUESTED** [review]

File: src/Module.Server/Modes/Conquest/CrpgConquestServer.cs (hunk 1 above)
Comment (MrMendo) [reply src/Module.Server/Modes/Conquest/CrpgConquestServer.cs:61]: starting the attackers at 3x multi ensured the equality of XP gained. Also thematically i thought this made sense, as defenders start to rack up multi instantly and get good rewards, while attackers always remained at 1x multi.
In my docs there is also an alternative of starting at 2x multi if that is too much.

File: src/Module.Server/Modes/Conquest/CrpgConquestServer.cs (hunk 2 above)
Comment (MrMendo) [reply src/Module.Server/Modes/Conquest/CrpgConquestServer.cs:450]: Fips and i felt that it would feel better for a casual mode to get rewards more often. Of course it would be the same to get the tick each minute, but this felt better to me.

File: src/Module.Server/Modes/Conquest/CrpgConquestServer.cs (hunk 1 above)
Comment (verdie-g) [reply src/Module.Server/Modes/Conquest/CrpgConquestServer.cs:61]: oooh ok that's the reason. Could you comment your choice in the code, I thought that was about when a guy gets rebalanced in another team.

File: src/Module.Server/Modes/Conquest/CrpgConquestServer.cs (hunk 2 above)
Comment (verdie-g) [reply src/Module.Server/Modes/Conquest/CrpgConquestServer.cs:450]: Hmm, not a big fan of having inconsistent rewards between ticks. I think that will just confuse people.

File: src/Module.Server/Modes/Conquest/CrpgConquestServer.cs (hunk 1 above)
Comment (MrMendo) [reply src/Module.Server/Modes/Conquest/CrpgConquestServer.cs:61]: In my opinion it's generally harder to attack than to defend and helping the attackers get some rewards at the start made the rewards more fair compared to defenders

Comment (verdie-g) [reply src/Module.Server/Modes/Conquest/CrpgConquestServer.cs:61]: Sure I'm good with that. Could you add a comment in the code please.

File: src/Module.Server/Modes/Conquest/CrpgConquestServer.cs (hunk 2 above)
Comment (MrMendo) [reply src/Module.Server/Modes/Conquest/CrpgConquestServer.cs:450]: You think it's confusing? I think it's the same as finding out how the multi system even works on conquest. IMO it's not much more complex than it already is. I really think that more ticks are more fun than big ticks per minute.
But if you think that is too much i can remove it again.

File: src/Module.Server/Modes/Conquest/CrpgConquestServer.cs (hunk 1 above)
Comment (MrMendo) [reply src/Module.Server/Modes/Conquest/CrpgConquestServer.cs:61]: Done

File: src/Module.Server/Modes/Conquest/CrpgConquestServer.cs (hunk 2 above)
Comment (verdie-g) [reply src/Module.Server/Modes/Conquest/CrpgConquestServer.cs:450]: Ah nvm I misunderstood the code, I thought the even tick could be less than the odd tick for defenders.

File: src/Module.Server/Rewards/CrpgRewardServer.cs [hunk 3]
```diff
@@ -135,7 +135,7 @@ public override void OnRemoveBehavior()
             return;
         }
 
-        bool lowPopulationServer = networkPeers.Length < 4;
```
Comment (verdie-g) [inline src/Module.Server/Rewards/CrpgRewardServer.cs:138]: To discard I suppose

**Review by verdie-g — CHANGES_REQUESTED** [review]

File: src/Module.Server/Rewards/CrpgRewardServer.cs (hunk 3 above)
Comment (MrMendo) [reply src/Module.Server/Rewards/CrpgRewardServer.cs]: Yeah.. committed the wrong file. oops

**Review by verdie-g — APPROVED** [review]
